tests/vision/utils_tests/metrics_test.py

```diff
@@ -8,38 +8,59 @@
 # along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
 # ----------------------------------------------------------------------------
 #
+from typing import Dict
+
 from hamcrest import assert_that, close_to, has_items, has_length
+from ignite.engine import Engine
+from ignite.metrics import Metric
```

Comment:
Aren't we getting rid of ignite metrics?

Reply:
it tests they make life easier